airbyte-integrations/bases/source-acceptance-test/unit_tests/test_spec.py

```diff
@@ -648,3 +649,105 @@ def test_additional_properties_is_true(connector_spec, expectation):
     t = _TestSpec()
     with expectation:
         t.test_additional_properties_is_true(connector_spec)
+
+
+@pytest.mark.parametrize(
+    "connector_spec, should_fail",
+    (
+        (
+            {
+                "connectionSpecification": {"type": "object", "properties": {"api_token": {"type": "string", "airbyte_secret": True}}}
```

Comment:
I suggest to add a case to check that the test is not failing for a property that is not part reserved keyword and is not marked as Airbyte secret.

Reply:
done